antarest/study/service.py

```diff
@@ -2379,6 +2394,12 @@ def unarchive_output(
         assert_permission(params.user, study, StudyPermissionType.READ)
         self._assert_study_unarchived(study)
 
+        output_path = Path(study.path) / "output" / output_id
+        if not is_output_archived(output_path):
```

Comment:
not clear to me. Maybe this is better: 
if output_path.exists():
__if not is_output_archived(output_path):
____raise OutputAlreadyUnarchived(output_id)
else:
__raise OutputNotFound(output_id)

Reply:
Yep

Reply:
In fact it won't work :/ because with the if in this order if an output is unarchived we'll go inside the OutputNotFound